crates/ruff_python_formatter/tests/snapshots/black_compatibility@py_311__pep_654.py.snap

```diff
@@ -1,240 +0,0 @@
----
```

Comment:
Why deleted?

Reply:
Hmm strange -- no idea! I will restore. Wonder what's going on here, maybe an instability in the formatting?